.github/workflows/ci.yaml

```diff
@@ -1,4 +1,4 @@
-name: ci
+name: Build
```

Comment:
I'm all in favor of the change. Will it break anything though?

Reply:
Nothing that I can think of but hopefully we don't find out the hard way hahahaha

Reply:
lolol only one way to find out